src/Libraries/CoreNodeModels/Input/Sliders.cs

```diff
@@ -295,7 +295,7 @@ protected override bool ShouldDisplayPreviewCore()
 
     [NodeName("Integer Slider")]
     [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
-    [NodeDescription("A slider that produces integer values.")]
+    [NodeDescription("IntSliderNodeDescription",typeof(Resources))]
```

Comment:
Please follow the pattern: `ClassNameNodeDescription` (and replace the `ClassName` with the actual class name), can we redo all these `NodeDescription` thing?